Checkr/src/components/organisms/PreAdverse/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+//import theme from '../../../components/themes/index';
+import Layout from '../../../components/templates/Layout/index';
+import Buttons from '../../../components/atoms/Button/index';
+import { useState } from 'react';
+import PreviewModal from '../PreviewModal';
```

Comment:
All the imports from libraries should be placed at the top and followed by components imports and lastly relative imports. 
Maintain this format across the application.